### PR TITLE
Fix Prometheus TLS configuration

### PR DESCRIPTION
**Pull request by howardjohn**
This was misleading/inaccurate

Please provide a description for what this PR is for.

And to help us figure out who should review this PR, please 
put an X in all the areas that this PR affects.

- [ ] Ambient
- [ ] Docs
- [ ] Installation
- [ ] Networking
- [ ] Performance and Scalability
- [ ] Extensions and Telemetry
- [ ] Security
- [ ] Test and Release
- [ ] User Experience
- [ ] Developer Infrastructure

### CODE REVIEW

File: content/en/docs/ops/integrations/prometheus/index.md [hunk 1]
```diff
@@ -124,10 +124,21 @@ To configure an existing Prometheus instance to scrape stats generated by Istio,
...
-However, the application metrics will follow whatever Istio configuration has been configured
-for the workload. In particular, if [Strict mTLS](/docs/tasks/security/authentication/authn-policy/#globally-enabling-istio-mutual-tls-in-strict-mode)
+However, the application metrics will follow whatever [Istio configuration](/docs/tasks/security/authentication/authn-policy) has been configured
+for the workload.
+
+* If you use `STRICT` mode, TLS must be configured. See below for TLS configuration.
+* If you use `PERMISSIVE` mode, the workload typically accepts TLS and cleartext. However, Prometheus cannot send the special variant of TLS Istio requires for `PERMISSIVE` mode. As a result, you must *not* configure TLS in Prometheus.
+* If you use `DISABLE` mode, 
+In particular, if [Strict mTLS](/docs/tasks/security/authentication/authn-policy/#globally-enabling-istio-mutual-tls-in-strict-mode)
 is enabled, then Prometheus will need to be configured to scrape using Istio certificates.
```
Comment (frankbu) [inline content/en/docs/ops/integrations/prometheus/index.md:134]:
```suggestion
* If you use `DISABLE` mode, no TLS configuration is required for Prometheus.
```

File: content/en/docs/ops/integrations/prometheus/index.md [hunk 2]
```diff
@@ -124,10 +124,21 @@ To configure an existing Prometheus instance to scrape stats generated by Istio,
...
 
+Note this this only applies to Istio-terminated TLS. If your application directly handles TLS:
+* `STRICT` mode is not supported, as Prometheus would need to send two layers of TLS which it cannot do.
+* `PERMISSIVE` mode and `DISABLE` mode should be configured the same as if Istio was not present.
+
+See [Understanding TLS Configuration](/docs/ops/configuration/traffic-management/tls-configuration/) for more information.
```
Comment (frankbu) [inline content/en/docs/ops/integrations/prometheus/index.md:140]: Maybe this part is better in a tip?
```suggestion
{{< tip >}}
Note this only applies to Istio-terminated TLS. If your application directly handles TLS:
* `STRICT` mode is not supported, as Prometheus would need to send two layers of TLS which it cannot do.
* `PERMISSIVE` mode and `DISABLE` mode should be configured the same as if Istio was not present.

See [Understanding TLS Configuration](/docs/ops/configuration/traffic-management/tls-configuration/) for more information.
{{< /tip >}}
```

**Review by ilrudie — COMMENTED** [review]
Overall this adds some good clarity. Have a few suggestions about wording but nothing major.

File: content/en/docs/ops/integrations/prometheus/index.md [hunk 3]
```diff
@@ -124,9 +124,20 @@ To configure an existing Prometheus instance to scrape stats generated by Istio,
 #### TLS settings
 
 The control plane, gateway, and Envoy sidecar metrics will all be scraped over cleartext.
-However, the application metrics will follow whatever Istio configuration has been configured
-for the workload. In particular, if [Strict mTLS](/docs/tasks/security/authentication/authn-policy/#globally-enabling-istio-mutual-tls-in-strict-mode)
-is enabled, then Prometheus will need to be configured to scrape using Istio certificates.
+However, the application metrics will follow whatever [Istio configuration](/docs/tasks/security/authentication/authn-policy) has been configured
```
Comment (frankbu) [inline content/en/docs/ops/integrations/prometheus/index.md:127]: As suggested by @ilrudie:
```suggestion
However, the application metrics will follow whatever [Istio authentication policy](/docs/tasks/security/authentication/authn-policy) has been configured
```

**Review by craigbox — APPROVED** [review]

**Pull request merged**